src/battle_script_commands.c

```diff
@@ -16404,25 +16416,24 @@ void BS_SetPledge(void)
 
 void BS_SetPledgeStatus(void)
```

Comment:
Is there no check here for if the status is already set? As far as I'm aware, the timer doesn't reset if the effect is already active (unless I'm misremembering).

Reply:
Am I understanding it correctly that a combined pledge move attack still happens but the timer isn't reset?

Reply:
You can confirm with an ingame test, but generally when a field effect is set the timer isn't reset to maximum value each time the attack is used.

Reply:
Confirmed on showdown and applied change.